client/src/index.css

```diff
@@ -1,7 +1,26 @@
+@import url("https://fonts.googleapis.com/css2?family=Rochester&display=swap");
+
+:root  {
+  --color-background: #fbf2f2;
+  --color-list-items: #f4cbad;
+  --color-single-items: #f4b7b7;
+  --color-icons: #d84064;
+  --color-footer-border: #ea8989;
+}
```

Comment:
great! using variables at an early stage makes it more easy to maintain a good overview. really need to do it as well before it is too late. haha